doc/source/reference/random/bit_generators/index.rst

```diff
@@ -12,8 +14,68 @@ setting the state, jumping or advancing the state, and for accessing
 
-Stable RNGs
-===========
+
+The included BitGenerators are:
+
+* `MT19937 <mt19937>` - The standard Python BitGenerator. Produces identical results to
+  Python using the same seed/state. Adds a `~mt19937.MT19937.jumped` function
+  that returns a new generator with state as-if ``2**128`` draws have been made.
+* `DSFMT <dsfmt>` - SSE2 enabled versions of the MT19937 generator.  Theoretically
+  the same, but with a different state and so it is not possible to produce a
+  sequence identical to MT19937. See the `dSFMT authors' page`_.
+* `Xoshiro256 <xoshiro256>` and `Xoshiro512 <xoshiro512>` - The most recently
+  introduced XOR, shift, and rotate generator. More information about these bit
+  generators is
+  available at the `xorshift, xoroshiro and xoshiro authors' page`_.
+* `ThreeFry <threefry>` and `Philox <philox>` - counter-based generators
+  capable of being advanced an arbitrary number of steps or generating
+  independent streams. See the `Random123`_ page for more details about this
+  class of bit generators.
+* `PCG32 <pcg32>` and `PCG64 <pcg64>` are permutation-congruential generators.
+  More information is available on the `dSFMT authors' page`_.
+
+.. _`dSFMT authors' page`: http://www.math.sci.hiroshima-u.ac.jp/~m-mat/MT/SFMT/
+.. _`PCG author's page`: http://www.pcg-random.org/
+.. _`xorshift, xoroshiro and xoshiro authors' page`:  http://xoroshiro.di.unimi.it/
+.. _`Random123`: https://www.deshawresearch.com/resources_random123.html
+
+Summary:
+
+============== ==================== ================ ======== ======= =====================
+BitGenerator   period               jumped distance  speed relative
+                                                     to dSFMT [1]_
+-------------- -------------------- ---------------- ---------------- ---------------------
+                                                     linux64  win32
+============== ==================== ================ ======== ======= =====================
+``MT19937``                          :math:`2^{128}`  0.5x     5x     Also used in python's
+                                                                      `random`
+``dSFMT``                            :math:`2^{128}`  1x       10x    Faster version of MT19937
+``Xoshiro256`` :math:`2^{256} - 1`  :math:`2^{128}`
+``Xoshiro256`` :math:`2^{1024} - 1` :math:`2^{512}`
+``ThreeFry``   :math:`2^{256} - 1`  :math:`2^{128}`                   counter-based
+``PHilox``     :math:`2^{256} - 1`  :math:`2^{128}`                   counter-based
+``PCG32``      :math:`2^{64}`       :math:`2^{32}`                     
+``PCG64``      :math:`2^{128}`      :math:`2^{64}`                     
+============== ==================== ================ ======== ======= =====================
+
+.. rubric:: Footnotes
+
+.. [1] More is better. As always with benchmarks, these are rough guides, your
+       experience may vary. Win32 is relative to linux64, ``dSFMT``
```

Comment:
This table, which is difficult to format in RST, needs to be filled in (or revamped or removed). The numbers in the speed column are made up. Should we include timings at all here, should they be relative to a single baseline or per-OS?

Reply:
I would do speed relative to MT19937, which is the current release default.